src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php

```diff
@@ -38,7 +38,7 @@ public function createService(Locator $serviceLocator)
         $debugbar->addCollector(new ConfigCollector($appConfig, 'ApplicationConfig'));
 
         // Db profiler
-        if ($serviceLocator->has($dbServiceName) && isset($config['db']['driver'])) {
```

Comment:
Anyway one checking enough. This config parameter spoils the impression about debugger. =))